api/tendermint/crypto/proof.pulsar.go

```diff
@@ -2919,7 +2919,7 @@ func (x *DominoOp) GetOutput() string {
 }
 
 // ProofOp defines an operation used for calculating Merkle root
-// The data could be arbitrary format, providing nessecary data
+// The data could be arbitrary format, providing necessary data
```

Comment:
How have you fixed these? Have you manually edited this file?

Reply:
Yes, GoLand IDE will give the hints, and I manually edit the files.

Reply:
I learned that it's generated by protoc just now. It's my fault. I will resolved later.

Reply:
That's what I thought 😁
Indeed please revert those tendermint proto generated typo.
If you want to fix them you should do it in the cometbft/cometbft repo
For the other, please modify the proto file and run make proto-gen

Reply:
Thank you for your guidance. I have revert those tendermint proto generated typos, since it's easier...😁